packages/create-app/templates/default-app/packages/app/src/setupTests.ts

```diff
@@ -1 +1,2 @@
 import '@testing-library/jest-dom';
+import 'cross-fetch/polyfill';
```

Comment:
For future: I feel that regardless of whether this is needed or not for this particular change, it is correct to have this here for new apps